modules/rabbitmq/src/main/java/org/testcontainers/containers/RabbitMqContainer.java

```diff
@@ -0,0 +1,100 @@
+package org.testcontainers.containers;
+
+import java.time.Duration;
+
+import org.jetbrains.annotations.NotNull;
+import org.testcontainers.containers.wait.strategy.LogMessageWaitStrategy;
+import org.testcontainers.containers.wait.strategy.WaitStrategy;
+
+
+/**
+ * This {@link Container} is based on the official RabbitMQ ({@code rabbitmq}) images from
+ * <a href="https://hub.docker.com/_/rabbitmq/">DockerHub</a>.
+ *
+ * @author Stefan Ludwig
+ */
+public class RabbitMqContainer extends GenericContainer<RabbitMqContainer> {
+
+    private static final String DEFAULT_LOG_MESSAGE_REGEX = ".*Server startup complete.*\n";
+    private static final int DEFAULT_STARTUP_TIMEOUT = 30;
+
+    public static final int RABBITMQ_PORT = 5672;
+
+    /**
+     * Creates a new {@link RabbitMqContainer} with the {@code 'rabbitmq:latest'} image.
+     */
+    public RabbitMqContainer() {
+        this("latest");
```

Comment:
could you please fix the version? We decided to use fixed versions instead of `latest` to avoid a breakage when the image gets updated

Reply:
done